src/module/comments.js

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable consistent-return */
+/* eslint import/no-cycle: 0 */
+// eslint-disable-next-line no-unused-vars
+// import { text } from '@fortawesome/fontawesome-svg-core';
```

Comment:
- [ ] Please let's avoid disabling linters. They help you stick to a coding style, catch errors and maintain code uniformity with your co-contributors 👍 
- [ ] Also remove the commented code on `line 4` as it takes up space and is of no use to your code execution.